src/executables/csql.c

```diff
@@ -913,7 +919,28 @@ csql_do_session_cmd (char *line_read, CSQL_ARGUMENT * csql_arg)
       break;
 
     case S_CMD_EDIT:		/* invoke system editor */
-      if (csql_invoke_system_editor () != CSQL_SUCCESS)
+      if (*argument == '\0')
+	{
+	  csql_Editor_format = false;
+	}
+      else if (!strcasecmp (argument, "format") || !strcasecmp (argument, "fmt"))
+	{
+	  csql_Editor_format = true;
+	}
+      else
+	{
+	  csql_Error_code = CSQL_ERR_FORMAT_OPTION;
+	  nonscr_display_error (csql_Scratch_text, SCRATCH_TEXT_LEN);
+	  break;
+	}
+
+      csql_return_status = csql_invoke_system_editor (csql_Editor_format);
+      if (csql_return_status == CSQL_ERR_FORMAT)
```

Comment:
Do you have any special purpose this statement form?

I mean, like killtran, checkpoint case usually compares its return value with expression in the if-statement.

` case S_CMD_CHECKPOINT:
if (csql_arg->sysadm && au_is_dba_group_member (Au_user))`
` case S_CMD_KILLTRAN:
if (csql_arg->sysadm && au_is_dba_group_member (Au_user))`

So, my opinion is not breaking the unity of the code form.
But If you have any reasons that they should be different in this code, could you tell me about that?

Reply:
I understand. I will modify it to be similar to the existing code.
Thank you.